solutions/adaptive-image/README.md

```diff
@@ -96,7 +96,7 @@ Intersection observer enabled:
 
 ## Breakpoints
 
-If you have some responsive layout where images could be resized in some cases, it's good to set a list of breakpoints to avoid exceed sized generation via Uploadcare CDN operations:
+If you have some responsive layout where images could be resized in some cases, it's good to set a list of breakpoints to exceed sized generation via Uploadcare CDN operations:
```

Comment:
IMHO, meaning is changed.

My suggestion:
>If you have some responsive layout where images could be resized in some cases, it's good to set a list of breakpoints to avoid exceed sizes generation via Uploadcare CDN operations:

Reply:
Yes, that sounds better. Added.